konnectivity-socks5-proxy/main.go

```diff
@@ -54,13 +54,15 @@ func NewStartCommand() *cobra.Command {
 	var connectDirectlyToCloudAPIs bool
 	var resolveFromGuestClusterDNS bool
 	var resolveFromManagementClusterDNS bool
+	var disableResolver bool
 
 	cmd.Flags().StringVar(&proxyHostname, "konnectivity-hostname", "konnectivity-server-local", "The hostname of the konnectivity service.")
 	cmd.Flags().IntVar(&proxyPort, "konnectivity-port", 8090, "The konnectivity port that socks5 proxy should connect to.")
 	cmd.Flags().IntVar(&servingPort, "serving-port", 8090, "The port that socks5 proxy should serve on.")
 	cmd.Flags().BoolVar(&connectDirectlyToCloudAPIs, "connect-directly-to-cloud-apis", false, "If true, traffic destined for AWS or Azure APIs should be sent there directly rather than going through konnectivity. If enabled, proxy env vars from the mgmt cluster must be propagated to this container")
 	cmd.Flags().BoolVar(&resolveFromGuestClusterDNS, "resolve-from-guest-cluster-dns", false, "If DNS resolving should use the guest clusters cluster-dns")
 	cmd.Flags().BoolVar(&resolveFromManagementClusterDNS, "resolve-from-management-cluster-dns", false, "If guest cluster's dns fails, fallback to the management cluster's dns")
+	cmd.Flags().BoolVar(&disableResolver, "disable-resolver", false, "If true, DNS resolving is disabled. Takes precedence over resolve-from-management-cluster-dns and resolve-from-management-cluster-dns")
```

Comment:
did you mean resolve-from-management-cluster-dns and resolve-from-guest-cluster-dns here?

Reply:
ugh, of course!
Fixed.